tests/Bridge/Laravel/Validation/ExtendedUrlRuleTest.php

```diff
@@ -0,0 +1,81 @@
+<?php
+declare(strict_types=1);
+
+namespace Tests\EoneoPay\Externals\Bridge\Laravel\Validation;
+
+use EoneoPay\Externals\Bridge\Laravel\Validator;
+use Illuminate\Translation\ArrayLoader;
+use Illuminate\Translation\Translator;
+use Illuminate\Validation\Factory;
+use Tests\EoneoPay\Externals\TestCase;
+
+/**
+ * @covers \EoneoPay\Externals\Bridge\Laravel\Validation\ExtendedUrlRule
+ */
+final class ExtendedUrlRuleTest extends TestCase
+{
+    /**
+     * Data provider for testing extended_url rule.
+     *
+     * @return mixed[]
+     */
+    public function getValidationData(): iterable
```

Comment:
I'd like to see a more exhaustive test of invalid urls:

https://github.com/symfony/validator/blob/master/Tests/Constraints/UrlValidatorTest.php#L215-L259